README.md

```diff
@@ -40,6 +40,7 @@
 - Data Model.
 - Processing data in models.
 - Validations and Model specs.
+- Setup controllers and basic routing.
 
 ## Getting Started
```

Comment:
- [ ] I got the beautiful error page below when I click on the issue link. Let's kindly fix the issue link to provide your contributors best experience :+1: 
![Screenshot from 2023-08-29 06-29-08](https://github.com/ClintonYounge/My-Blog-App/assets/63087491/fda56f63-509c-409d-9f62-9b1966d906ab)

Reply:
I'm not quite sure how to fix this. The link's working for me. Maybe it's a setting thing.